utilities/write_batch_with_index/write_batch_with_index_test.cc

```diff
@@ -2766,6 +2767,148 @@ TEST_P(WriteBatchWithIndexTest, WideColumnsBatchAndDB) {
   ASSERT_FALSE(iter->Valid());
 }
 
+TEST_P(WriteBatchWithIndexTest, GetEntityFromBatch) {
+  ASSERT_OK(OpenDB());
+
+  // No base value, no merges => NotFound
+  {
+    constexpr char key[] = "a";
+
+    PinnableWideColumns result;
+    ASSERT_TRUE(
+        batch_->GetEntityFromBatch(db_->DefaultColumnFamily(), key, &result)
+            .IsNotFound());
+  }
+
+  // No base value, with merges => MergeInProgress
+  {
+    constexpr char key[] = "b";
+    constexpr char merge_op1[] = "bv1";
+    constexpr char merge_op2[] = "bv2";
+
+    ASSERT_OK(batch_->Merge("b", merge_op1));
+    ASSERT_OK(batch_->Merge("b", merge_op2));
```

Comment:
QQ to confirm my own understanding.

For the case of `Merge` -> `Merge` -> `Put`, the latest put will overwrite the previous two merges and `GetEntityFromBatch` will return OK, no longer MergeInProgress, right?

Reply:
Yep, that's correct.